pdc_client/plugins/rrt_image_tests.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+import json
+
+from pdc_client.plugin_helpers import (PDCClientPlugin,
+                                       extract_arguments,
+                                       add_parser_arguments)
+
+
+def print_build_image_rtt_list(rtt_tests):
+    fmt = '{0:<10} {1:40} {2:10} {3:6}'
+    for rtt_test in rtt_tests:
+        print fmt.format(rtt_test['id'],
+                         rtt_test['build_nvr'],
+                         rtt_test['format'],
+                         rtt_test['test_result'])
+
+
+class BuildImageRttTest(PDCClientPlugin):
+    def register(self):
+        self.set_command('build-image-rtt-tests')
+        list_parser = self.add_action('list', help='list all build image rtt tests')
+        add_parser_arguments(list_parser, {'build_nvr': {},
+                                           'image_format': {},
+                                           'test_result': {}},
+                             group='Filtering')
+        list_parser.set_defaults(func=self.list_build_image_rrt_tests)
+
+        info_parser = self.add_action('info', help='display details of a build image rtt tests')
+        info_parser.add_argument('build_nvr', metavar='BUILD_NVR')
+        info_parser.add_argument('image_format', metavar='IMAGE_FORMAT')
+        info_parser.set_defaults(func=self.build_image_rrt_tests_info)
+
+        update_parser = self.add_action('update', help='Partial update an exist' +
+                                        'build image rtt tests by test_result')
```

Comment:
If you think there are too many words in one line, you can 
update_parser = self.add_action('update', help='Partial update an existing \
                                                     build image rtt tests by test_result')